frontend/public/locales/fr/apply-adult-child.json

```
@@ -561,16 +561,16 @@
     "return-btn-link": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires.html"
   },
   "partner-information": {
-    "page-title": "Renseignements sur l'époux/épouse ou le conjoint/conjointe de fait",
-    "provide-sin": "Fournissez les renseignements sur votre époux/épouse ou conjoint/conjointe de fait actuel tels qu'ils figurent sur sa carte ou lettre d'assurance sociale (NAS).",
+    "page-title": "Renseignements sur l'époux/l'épouse ou le conjoint/la conjointe de fait",
+    "provide-sin": "Fournissez les renseignements sur votre époux/épouse ou conjoint/conjointe de fait actuel/actuelle tels qu'ils figurent sur sa carte ou sa lettre de numéro d'assurance sociale (NAS).",
     "required-information": "Ces renseignements sont nécessaires pour calculer le revenu familial net rajusté. Votre époux/épouse ou conjoint/conjointe de fait doit présenter sa propre demande au Régime canadien de soins dentaires.",
     "sin": "Numéro d'assurance sociale (NAS)",
     "date-of-birth": "Date de naissance",
     "last-name": "Nom de famille",
     "first-name": "Prénom",
-    "single-legal-name": "(FR) If your spouse or common-law partner has a single legal name",
-    "name-instructions": "Si vous utilisez un seul nom légal, veuillez indiquer ce nom dans les DEUX champs «\u00a0Prénom\u00a0» et «\u00a0Nom de famille\u00a0».",
-    "confirm-checkbox": "Je confirme que mon époux/épouse ou mon conjoint/conjointe de fait est au courant et d'accord que ses renseignements personnels soient communiqués.",
+    "single-legal-name": "Si votre époux/épouse ou conjoint/conjointe de fait a un seul nom légal",
+    "name-instructions": "Si la personne utilise un seul nom légal, veuillez indiquer ce nom dans les DEUX champs «\u00a0Prénom\u00a0» et «\u00a0Nom de famille\u00a0»",
+    "confirm-checkbox": "Je confirme que mon époux/épouse ou conjoint/ conjointe de fait est au courant de la communication de ses renseignements personnels et qu'il/elle y consent.",
```

Is the space a mistake or it matches Figma?
`conjoint/ conjointe`

```suggestion
    "confirm-checkbox": "Je confirme que mon époux/épouse ou conjoint/conjointe de fait est au courant de la communication de ses renseignements personnels et qu'il/elle y consent.",
```

---

that's just how it copied over from figma :shrug: 
Man i hate figma so much

---

It's what I thought. It's all good now. 😃